packages/font-icons/scss/index.scss

```diff
@@ -1,6 +1,6 @@
-@import "_font.scss";
-@import "_variables.scss";
-@import "_icon-list.scss";
+@use "_font.scss" as *;
+@use "_variables.scss" as *;
```

Comment:
we should probably `@forward` the variables here in order to allow customization of the variables